tools/testing/target_determination/heuristics/interface.py

```diff
@@ -0,0 +1,86 @@
+from abc import abstractmethod
```

Comment:
typo?

Reply:
Surprisingly, that is the legit package name. Stands for "Abstract Base Class"